samples/templates/express-direct-auth/web-server/views/errors.mustache

```diff
@@ -1,5 +1,5 @@
 {{#hasError}}
-  <div class="ui negative message">
+  <div id="error-banner" class="ui negative message">
```

Comment:
I saw `okta-oidc-tck` repo is using `data-se` as e2e selectors. We probably can also follow the same pattern here, as class and id in html are subject to change in the future. 
But as it's a sample app, I don't have too strong opionion on the selector pattern.

Reply:
`data-se` gives e2e tests their own namespace, has obvious technical advantages. BUT it is a sample app, and `data-se` is something we made up and the user cannot search to find a meaning for this unless we document it.  I lean towards using `id` for the sample apps, since it is unique and requires no further explanation.